config.py

```diff
@@ -31,18 +31,18 @@
 # or
 # >>> study_path = '/Users/sophie/repos/ExampleData/'
 
-study_path = sample.data_path()
+study_path = '/home/stefanappelhoff/Downloads/eeg_matchingpennies'
 
 # ``subjects_dir`` : str
 #   The ``subjects_dir`` contains the MRI files for all subjects.
 
-subjects_dir = os.path.join(study_path, 'subjects')
+subjects_dir = os.path.join(study_path)
 
-# ``meg_dir`` : str
-#   The ``meg_dir`` contains the MEG data in subfolders
+# ``meeg_dir`` : str
+#   The ``meeg_dir`` contains the MEG or EEG data in subfolders
 #   named my_study_path/MEG/my_subject/
 
-meg_dir = os.path.join(study_path, 'MEG')
+meeg_dir = os.path.join(subjects_dir, 'eeg')
```

Comment:
Just assume BIDS structure and get rid of this, no?

Reply:
yes !